ui/tx/skeletons/TxInternalsSkeleton.tsx

```
@@ -0,0 +1,20 @@
+import { Skeleton, Box, VStack, HStack } from '@chakra-ui/react';
```

ESLint

[error] 'Skeleton' import from '@chakra-ui/react' is restricted. Please use corresponding component or hook from "toolkit" instead

(no-restricted-imports)